src/main/java/frc/robot/RobotContainer.java

```diff
@@ -70,7 +80,7 @@ public RobotContainer() {
    */
   private void configureBindings() {
     driverXbox.a().onTrue((Commands.none()));
-    driverXbox.b().onTrue(Commands.none());
+    driverXbox.b().onTrue((Commands.none()));
```

Comment:
```suggestion
    driverXbox.b().onTrue(Commands.none());
```
remove in the line above too